src/components/dash/header.tsx

```diff
@@ -1,22 +1,34 @@
 import React from 'react'
-import {View, Text, StyleSheet} from 'react-native'
+import {View, Text, StyleSheet, ToastAndroid} from 'react-native'
 import { Appbar } from 'react-native-paper'
+import FontAwesome from 'react-native-vector-icons/FontAwesome'
 import { DrawerActions, useNavigation } from '@react-navigation/native'
 import {useStores} from '../../store'
 import { useObserver } from 'mobx-react-lite'
 
 export default function Header() {
   const navigation = useNavigation()
-  const {details} = useStores()
+  const {details, ui} = useStores()
+
+  const checkConnectionHandler = () => {
+    const connected = ui.connected ? 'Connected node' : 'Disconnected noe';
```

Comment:
noe -> node

Reply:
@pitoi sorry about that! should be fixed now!